### PR TITLE
Go remote mode fixes

### PR DESCRIPTION
**Pull request by apmattil**
fixes issue #199

### CODE REVIEW

**Comment by apmattil** [comment]
travis test seem to fail to:
```
- Installing [ 4/14] lsp-java (latest)... downloadingDependency lsp-java failed to install: Package ‘spinner-1.7.3’ is unavailable
```
not relaited to this PR.

File: dap-go.el [hunk 1]
```diff
@@ -53,29 +53,22 @@
...
-                                                                             (process-attributes pid)))
-                                                                       (format "%6d %-30s %s" pid comm user)))
-                                                                   nil t))))
-                )))
+               ("remote" (dap--put-if-absent conf :program (f-dirname (buffer-file-name)))
+		         (dap--put-if-absent conf :host (read-string "enter host:" "127.0.0.1"))
+		         (dap--put-if-absent conf :port (string-to-number (read-string "Enter port: " "2345"))))
+               ("attach"
+                (dap--put-if-absent conf :program (f-dirname (buffer-file-name)))
+                (dap--put-if-absent conf :processId (string-to-number (read-string "Enter pid: " "2345")))
```
Comment (yyoncho) [inline dap-go.el]: can you put the closing brackets on the same line?

**Comment by yyoncho** [comment]
Thank you!

**Comment by apmattil** [comment]
never updated pull request .. It was easy ;)

**Pull request merged**